.github/workflows/release_notes.yml

```diff
@@ -0,0 +1,39 @@
+name: Check Release Notes
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
+      - labeled
+      - unlabeled
+    paths:
+      - "**.py"
+      - "pyproject.toml"
+      - "!.github/**/*.py"
+      - "!rest_api/**/*.py"
```

Comment:
It is not clear to me why we don't want to check the REST API files...

Reply:
The `rest_api` is not part of the Haystack package and release process